packages/cart/src/models/index.ts

```diff
@@ -1,2 +1,11 @@
-export { default as Cart } from "./cart";
+export { default as Address } from "./address"
+export { default as AdjustmentLine } from "./adjustment-line"
+export { default as Cart } from "./cart"
+export { default as LineItem } from "./line-item"
+export { default as LineItemAdjustmentLine } from "./line-item-adjustment-line"
+export { default as LineItemTaxLine } from "./line-item-tax-line"
+export { default as ShippingMethod } from "./shipping-method"
+export { default as ShippingMethodAdjustmentLine } from "./shipping-method-adjustment-line"
+export { default as ShippingMethodTaxLine } from "./shipping-method-tax-line"
+export { default as TaxLine } from "./tax-line"
```

Comment:
q: since TaxLine and LineItem are not actual models, should we remove them from here?

Reply:
Yes, good catch!